contributing.md

```diff
@@ -0,0 +1,106 @@
+# Enarx RFCs
+
+## Contributing
+
+### Do you need an RFC?
+
+Use an RFC to advocate substantial changes to the Aries ecosystem, where
```

Comment:
I think we want it to say "Enarx" here. : )

Reply:
Indeed! Thanks.